packages/contracts/contracts/factory/PluginRepoFactory.sol

```diff
@@ -42,21 +42,19 @@ contract PluginRepoFactory {
     /// @notice Creates and registers a named `PluginRepo` and publishes an initial version.
     /// @dev The initial owner of the new PluginRepo is `address(this)`, afterward ownership will be transfered to the address `_maintainer`.
     /// @param _name The plugin repository name.
-    /// @param _initialSemanticVersion The semantic version for the new plugin repository version.
     /// @param _pluginSetup The plugin factory contract associated with the plugin version.
     /// @param _contentURI The external URI for fetching the new version's content.
     /// @param _maintainer The plugin maintainer address.
-    function createPluginRepoWithVersion(
+    function createPluginRepoWithContractAndContentURI(
```

Comment:
Just a suggestion...
The wording "Contract" is general, perhaps a better naming would be `createPluginRepoWithFirstVersion`.

Reply:
done.